packages/exchange-connector/src/exchangeDownloader.ts

```diff
@@ -12,21 +12,24 @@ import fetch, { Response } from "node-fetch";
 import path from "path";
 
 import { RestApi, FileInfo, Categories } from "./exchangeTypes";
+import { sdkLogger } from "../../core/src";
 
 const DEFAULT_DOWNLOAD_FOLDER = "download";
 const ANYPOINT_BASE_URI = "https://anypoint.mulesoft.com/exchange/api/v2";
+const ANYPOINT_BASE_URI_WITHOUT_VERSION =
+  "https://anypoint.mulesoft.com/exchange";
 
 export function downloadRestApi(
```

Comment:
Just a suggestion, but this could be simplified a bit...
```
export function async downloadRestApi(
  restApi: RestApi,
  destinationFolder: string = DEFAULT_DOWNLOAD_FOLDER
): Promise<void | Response> {

    if (!restApi.fatRaml?.createdDate) {
      sdkLogger.warn(
        `Failed to download '${restApi.name}' RAML as Fat RAML download information is missing.`,
        `Please download it manually from ${ANYPOINT_BASE_URI_WITHOUT_VERSION}/${restApi.assetId} and update the relevant details in apis/api-config.json`
      );
      return;
    }

    ensureDirSync(destinationFolder);
    const zipFilePath = path.join(destinationFolder, `${restApi.assetId}.zip`);
    const response = await fetch(restApi.fatRaml.externalLink);
    writeFileSync(zipFilePath, response.body);
    return response;
}
```

Reply:
I like this proposal